src/views/Layout.jsx

```diff
@@ -1,4 +1,4 @@
-import { Outlet } from 'react-router-dom';
+import { NavLink, Outlet } from 'react-router-dom';
```

Comment:
This change is from the other team's commit and should not be here. We need to see how to handle these cases since merging is not the way to go here but rebase